pyxem/generators/diffraction_generator.py

```diff
@@ -136,7 +150,8 @@ def calculate_ed_data(self,
     def calculate_profile_data(self, structure,
                                reciprocal_radius=1.0,
                                magnitude_tolerance=1e-5,
-                               minimum_intensity=1e-3):
+                               minimum_intensity=1e-3,
+                               scattering_params='lobato'):
```

Comment:
Not sure you need this as an argument of this method. It gets overwritten by whatever you specified in the `__init__` for `Diffraction_Generator`